rfcs/2022-12-09-541-reload-api.md

```diff
@@ -53,6 +53,46 @@ environment variables will not be interpolated. The HTTP response will
 synchronously indicate the success or failure of the triggered reload,
 including any error messages scoped by component name.
 
+An example request would be as follows:
+
+```
+POST /config
+
+{
+  "sources": {
+    "in": {
+      "type": "stdin"
+    }
+  },
+  "sinks": {
+    "out": {
+      "inputs": [
+        "in"
+      ],
+      "type": "console",
+      "encoding": {
+        "codec": "text"
+      }
+    }
+  }
+}
+```
+
+Such a request would receive a simple `200 OK` response when the reload
+succeeded, or a 400-level response with a JSON structure in the body describing
+any errors that occurred:
```

Comment:
Trying to think about any other situations we may want to be aware of...

Right now the only thing that comes to mind is attempting to reload a config which is identical to the running config - would that just be an error? It wasn't really successful, but not necessarily an error - does it merit a specific code?

Reply:
We shouldn't need any different handling for that than exists today. I don't see anything wrong with that simply returning success.